dfinity/nns-dapp
# Add index canister dashboard link to the wallet popup

# Motivation

In the "more" popup for tokens, there should be two links: one to the ledger canister on the dashboard and one to the index canister. Here, we add the index canister link for the NNS account.

# Changes

- Remove the `noLabel` prop from `LinkToDashboardCanister`. Since different text is used in the component, the `noLabel` parameter is unnecessary as its value can be determined by the presence or absence of the `label` parameter.
- Add link to the index canister.
- Adjust the alignment of `a.button.with-icon` in `LinkToDashboardCanister` to ensure both links are properly aligned when rendered together.

# Tests

- Add no label test.
- Add "should have canister links in 'more' popup".
- Locally:
<img width="281" alt="image" src="https://github.com/user-attachments/assets/bbc13f7d-b296-42c5-946f-89fc8b0479a4">


# Todos

Not necessary.
